roofit/xroofit/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+ROOT_STANDARD_LIBRARY_PACKAGE(RooFitxRooFit
```

```suggestion
ROOT_STANDARD_LIBRARY_PACKAGE(RooFitXRooFit
```
Can we capitalize the X to be consistent with the namespace?